src/plugins/azgenerator/templates/topcommon/CliReport.ts

```diff
@@ -7,162 +7,296 @@ import { CodeModelAz } from "../../CodeModelAz"
 import { ParameterLocation, SchemaType } from "@azure-tools/codemodel";
 import { isNullOrUndefined } from "util";
 
-export function GenerateAzureCliReport(model: CodeModelAz) : string[] {
+export function GenerateAzureCliReport(model: CodeModelAz): string[] {
     var output: string[] = [];
```

Comment:
1. May need rebase master as I move the file to templates/CliReports.ts. 
2. On the other hand, Could you use TemplateBase as base class?